src/main/resources/application.yml

```diff
@@ -13,3 +13,8 @@ spring:
     properties.hibernate:
       format_sql: true
       show_sql: ${SHOW_SQL:true}
+
+application:
+  h2-column-encryption:
```

Comment:
Prefixing this h2 is an implementation detail.  Any database could be used.  It's a nitpick, this could be named better.